.htaccess

```diff
@@ -1,23 +1,25 @@
 # Original
 # If you modify this file then change the above line to: # Modified
+
 <IfModule mod_rewrite.c>
     RewriteEngine On
+
     # Certain hosts may require the following line.
     # If vanilla is in a subfolder then you need to specify it after the /.
     # (ex. You put Vanilla in /forum so change the next line to: RewriteBase /forum)
     # RewriteBase /
 
-    RewriteCond %{QUERY_STRING} ^p=/?([^&]+)(&(.+))?$
-    RewriteRule ^index\.php index.php/%1?%3 [E=X_REWRITE:1,L]
+    RewriteCond %{QUERY_STRING} ^p=/?([^&]+)(&([^?]+))?$
+    RewriteRule ^index\.php %1?%3 [E=X_REWRITE:1,L]
 
     # The basic rewrite rule.
     RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteCond %{REQUEST_FILENAME} !-d
```

Comment:
My guess is that it was removed because if a directory named categories is present on disk we it would prevent us from seeing http://www.example.com/categories